packages/contract-deposit/README.md

```diff
@@ -0,0 +1,34 @@
+# Contract deposit Tutorial
+
+`Contract deposit` is a simple demo of Arbitrum's L1-to-L2 deposit sending from a l1 contract
+
+It will deposit funds to l2 via a l1 contract, and because the inbox contract will alias the sender address if it is a contract,
+this tutorial shows how to control the alias address on l2 via its l1 contract address.
+
+This tutorial is just used to simply show how l1 contract controls its l2 alias and transfer its l2 alias funds to other address, if you want to use it as a tool, we suggest you to use our [Funds Recovery Tool](https://github.com/OffchainLabs/arbitrum-funds-recovery-tool).
+
```

Comment:
```suggestion
This tutorial demonstrates depositing funds to L2 using an L1 contract. Since the Inbox contract changes the sender address if it's a contract [(a.k.a address aliasing)](https://docs.arbitrum.io/how-arbitrum-works/arbos/l1-l2-messaging#address-aliasing), it explains how to manage the new L2 address using the L1 contract address. It's a basic example of how an L1 contract controls its L2 alias and transfers its alias funds to another address. For practical use, we recommend our [funds recovery tool]((https://github.com/OffchainLabs/arbitrum-funds-recovery-tool)).
```